.github/workflows/update-script.js

```diff
@@ -0,0 +1,158 @@
+const https = require('https');
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+// Run from the root of the project for local testing: node --env-file=.env .github/workflows/update-script.js
+const localExecution = process.env.LOCAL_RUN || false;
+const localRunConfigs = {
+  branch: process.env.LOCAL_RUN_BRANCH || 'update-imslib',
+  title: process.env.LOCAL_RUN_TITLTE || '[AUTOMATED-PR] Update imslib.min.js dependency',
+  path: process.env.LOCAL_RUN_SCRIPT || 'https://auth.services.adobe.com/imslib/imslib.min.js',
+  scriptPath: process.env.LOCAL_RUN_SCRIPT_PATH || './libs/deps/imslib.min.js',
+  origin: process.env.LOCAL_RUN_ORIGIN || 'origin',
+};
+
+const getPrDescription = ({ branch, scriptPath }) => `## Description
+Update ${scriptPath} to the latest version
+## Related Issue
+Resolves: NO TICKET - AUTOMATED CREATED PR.
```

Comment:
Can we link to a wiki page or [discussion](https://github.com/orgs/adobecom/discussions/1461)? It should be easier for folks to understand the details.